content/en/metrics/metrics-without-limits.md

```diff
@@ -22,21 +22,43 @@ algolia:
 
 This page identifies key components of Metrics without Limits™ that can help you manage your custom metrics volumes within your observability budget.
 
 ### Configuration of tags
 
-Click on any metric name to open its details sidepanel. Then click **Manage Tags** -> **"Include Tags..."** to configure the tags you'd like to remain as queryable on dashboards and monitors. The tag configuration modal pre-populates by default with an allowlist of tags that have been actively queried on dashboards, notebooks, monitors, and through API in the past 30 days (colored in blue with an icon). You can also include your own additional tags. Before selecting **Save**, an estimated new volume of indexed custom metrics that results from this potential tag configuration is displayed.
+#### Allowlist of tags 
+1. Click any metric name to open its details side panel. 
+2. Click **Manage Tags** -> **"Include Tags..."** to configure the tags you'd like to remain as queryable in dashboards, notebooks, monitors, and other Datadog products.
+3. Define your allowlist of tags. 
+By default, the tag configuration modal pre-populates with a Datadog recommended allowlist of tags that have been actively queried on dashboards, notebooks, monitors, or via the API in the past 30 days. Our recommendations are distinguished with an icon. 
+4. Review the *Estimated New Volume* of indexed custom metrics that results from this potential tag configuration.
+5. Click **Save**.
 
-{{< img src="metrics/mwl_tag_config.mp4" alt="Configuration of Tags" video=true >}}
+{{< img src="metrics/mwl_example_include_tags-compressed.mp4" alt="Configuration of Tags with Allowlist" video=true style="width:100%" >}}
 
+There are APIs available to [create][2], [edit][3], and [delete][4] a tag configuration along with an [API][5] for estimating the potential impact of your configuration.
 
-There are also APIs available so you can [create][2], [edit][3], and [delete][4] a tag configuration. There is also an [API][5] for estimating the potential impact of your configuration.
+#### Blocklist of tags 
+1. Click on any metric name to open its details side panel.
+2. Click **Manage Tags** -> **"Exclude Tags…"** to drop tags you don't want to query. 
+3. Define your blocklist of tags. These tags will **not** be queryable on dashboards and monitors. Tags that have been actively queried on dashboards, notebooks, monitors, and through the API in the past 30 days will be distinguished with an icon.
```

Comment:
```suggestion
3. Define your blocklist of tags. These tags are **not** queryable on dashboards and monitors. Tags that have been actively queried on dashboards, notebooks, monitors, and through the API in the past 30 days will be distinguished with a graph line icon.
```

Reply:
I think it makes more sense here to say these tags "will not be queryable" rather than "these tags are not queryable" as we are talking about setting the blocklist, not the list being set already.

Reply:
How about this? We try to avoid future tense in docs
```suggestion
3. Define your blocklist of tags. Tags defined in the blocklist are **not** queryable on dashboards and monitors. Tags that have been actively queried on dashboards, notebooks, monitors, and through the API in the past 30 days are distinguished with a graph line icon.
```